test/helpers.js

```diff
@@ -45,6 +46,12 @@ MockIrcd.prototype.getIncomingMsgs = function() {
     return this.incoming.filter(function(msg) { return msg; });
 };
 
-module.exports = function(port, encoding) {
+module.exports.getFixtures = function(testSuite) {
+    var raw = fs.readFileSync('test/data/fixtures.json');
```

Comment:
Instead of using `fs.readFileSync` you can just require json files.

```javascript
var fixtures = require('./data/fixtures.json');
module.exports.getFixtures = function(testSuite) {
  return fixtures[testSuite];
};
```